report_thesis/src/sections/problem_definition.tex

```diff
@@ -1,28 +1,22 @@
 
-To evaluate the performance of these models, we will use \gls{rmse} as a proxy for accuracy, defined by the equation:
+Subsequent missions, such as the \gls{mer} mission in 2003 and the \gls{msl} mission in 2012, sought to investigate whether Mars ever had the conditions to support life. 
+The Curiosity rover, part of the \gls{msl} mission, is equipped with the \gls{chemcam} instrument, which uses \gls{libs} to gather spectral data from geological samples on Mars~\cite{wiensChemcam2012}.
 
-\begin{equation}
-	RMSE = \sqrt{\frac{1}{n} \sum_{i=1}^{n} (y_i - \hat{y}_i)^2},
-\end{equation}
+\gls{libs} uses a laser to ablate surface material and generate a plasma plume, which emits light captured by spectrometers. 
+The resulting spectra consist of emission lines associated with the concentration of specific elements, serving as a multi-dimensional fingerprint of the sample's elemental composition~\cite{cleggRecalibrationMarsScience2017}.
 
-where $y_i$ represents the actual values, $\hat{y}_i$ the predicted values, and $n$ the number of observations.
-We use \gls{rmse} as a measure of accuracy to quantify the difference between the predicted and actual values of the major oxides in the samples.
+\subsection{Problem Formulation}
+Predicting major oxide compositions from \gls{libs} data presents significant computational challenges, including high dimensionality, non-linearity, multicollinearity, and the phenomenon known as matrix effects~\cite{andersonImprovedAccuracyQuantitative2017}.
+Furthermore, the high cost of data collection often results in small datasets, complicating the task of building accurate and robust models.
 
-To address robustness, we will consider the standard deviation of prediction errors across each oxide and test instance, defined as:
+Our problem definition builds upon the definitions provided in \citet{p9_paper}.
 
-\begin{equation}
-	\sigma_{error} = \sqrt{\frac{1}{n-1} \sum_{i=1}^{n} (e_i - \bar{e})^2},
-\end{equation}
+% TODO: Write data definitions here; define problem formally.
 
-where $e_i = y_i - \hat{y}_i$ and $\bar{e}$ is the mean error.
-We use the standard deviation of prediction errors to measure the variability of the prediction errors across each oxide and test instance, which provides a measure of the robustness of the model's predictions.
-
-\textit{Problem Definition:} Insert problem definition here.
+\textbf{Problem Definition:} This thesis aims to address the challenges in predicting major oxide compositions from \gls{libs} data by developing machine learning models that improve the accuracy and robustness of these predictions. 
+We will investigate various techniques to handle the high dimensionality, non-linearity, and small dataset size inherent in this problem, and evaluate model performance using appropriate metrics, which will be discussed in detail in Section~\ref{sec:methodology}.
```

Comment:
Should we also tone it down here and instead say something more akin to "explore" instead of "address"?

Reply:
Will do this in the designated problem def task